### ISSUE TITLE
How do I inverse a matrix?

### ISSUE THREAD

**Issue by aligajani** [powder96/numbers.php#5]

**Comment by Dakkers** [comment]
doesn't seem to be a part of Matrix.php so I don't think you can :\

**Comment by aligajani** [comment]
I suppose this is a basic feature of any Matrix math library :(

On Fri, Jun 27, 2014 at 10:45 PM, Dakota St. Laurent <
notifications@github.com> wrote:

> doesn't seem to be a part of Matrix.php so I don't think you can :\
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/powder96/numbers.php/issues/5#issuecomment-47403819.

## 

Ali Gajani
Founder at Mr. Geek
www.mrgeek.me
www.aligajani.com

**Comment by Dakkers** [comment]
yeah, I'm surprised it's not a part of it, especially considering this is a fork of Numbers.js which had it...hmm. maybe I'll take a stab at it.

**Comment by aligajani** [comment]
Sure. Any good PHP library that does it?

Or perhaps someone can add to this.

On Saturday, June 28, 2014, Dakota St. Laurent notifications@github.com
wrote:

> yeah, I'm surprised it's not a part of it, especially considering this is
> a fork of Numbers.js which had it...hmm. maybe I'll take a stab at it.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/powder96/numbers.php/issues/5#issuecomment-47418137.

## 

Ali Gajani
Founder at Mr. Geek
www.mrgeek.me
www.aligajani.com

**Comment by Dakkers** [comment]
I'm unsure - I've never done PHP, I'm just a collab of Numbers.js so i was thinking of adding to this repo too.

**Comment by GordonLesti** [comment]
@aligajani I'm working on https://github.com/NumPHP/NumPHP, maybe you would like to try it?

**Comment by ritvikgautam** [comment]
Updated and created a pull request #7 to implement inverse of a matrix.
